src/main/java/uk/gov/companieshouse/web/pps/util/PenaltyUtils.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.companieshouse.web.pps.util;
+
+public class PenaltyUtils {
+
+    private PenaltyUtils() {
+    }
+
```

Comment:
Don't know if I should be keeping this empty method in here, I was getting a warning on IntelliJ and it suggested adding this to get rid of it. Let me know if this needs changed or any suggestions instead of having this

Reply:
Keep it in, IntelliJ know best :) It's a private constructor.

Reply:
Another point, when in doubt, check Sonar either locally or the web report for the PR -- https://code-analysis.platform.aws.chdev.org/dashboard?id=uk.gov.companieshouse%3Apenalty-payment-web&pullRequest=178